static/docs/tutorials/deep/define-ml-pipeline.md

```diff
@@ -168,8 +168,8 @@ $ dvc run -d data/Posts.xml.zip -o data/Posts.xml \
 Running command:
   unzip data/Posts.xml.zip -d data/
 Archive:  data/Posts.xml.zip
+    creating data/Posts.xml/
```

Comment:
Hm, where did this come from? I'm not seeing it from `unzip data/Posts.xml.zip -d data/`.

Reply:
@efiop I tested by creating a custom zip file. Wait, I will check with `Posts.xml.zip` too.